### PR TITLE
Migrate Query Source to React: resizable areas (v2)

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Refactor

## Description

- [x] Resizable areas
- [x] Collapse/expand
  - [x] Animation
- [x] Collapse/expand using shortcuts

Replaces getredash/redash#4501

## Related Tickets & Documents

getredash/redash#4429

### CODE REVIEW

**Comment by kravets-levko** [comment]
:thinking: very interesting, I'll check ASAP

**Comment by gabrieldutra** [comment]
The resize areas are now working smoothly for both mobile and desktop :rocket:. One small thing missing -> The scrollbox for the results in desktop version:
image

File: client/app/components/dashboards/DashboardGrid.jsx [hunk 1]
```diff
@@ -9,6 +9,7 @@ import cfg from "@/config/dashboard-grid-options";
 import AutoHeightController from "./AutoHeightController";
 import { WidgetTypeEnum } from "@/services/widget";
 
+import "react-resizable/css/styles.css";
```
Comment (gabrieldutra) [inline client/app/components/dashboards/DashboardGrid.jsx]: Why import it specifically in here?

Comment (kravets-levko) [reply client/app/components/dashboards/DashboardGrid.jsx]: Right, it's not needed :+1: Forgot to remove after some experiments

**Pull request merged**